release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1726,6 +1726,43 @@ In the following tables, features are marked with the following statuses:
 
 * A regression in the behaviour of `libreswan` caused some nodes with IPsec enabled to lose communication with pods on other nodes in the same cluster. To resolve this issue, consider disabling IPsec for your cluster. (link:https://issues.redhat.com/browse/OCPBUGS-43713[*OCPBUGS-43713*])
 
+* There is a known issue in {product-title} {product-version} which causes the cluster's masquerade subnet to be set to `169.254.169.0/29` if the `ovnkube-node` daemon set is deleted. When the masquerade subnet is set to `169.254.169.0/29`, `UserDefinedNetwork` custom resources (CRs) cannot be created.
++
+[NOTE]
+====
+* If your masquerade subnet has been configured at Day 2 by making changes to the `network.operator` CR, it will not be reverted to `169.254.169.0/29`.
```

Comment:
@stevsmit Do we have affirmative info on this one? I have not tested with Day 2  configuration. Will run a test and confirm.